regex-assembly/942340.ra

```diff
@@ -3,26 +3,155 @@
 
 ##!+ i
 
+##!> define space_or_word \s*[\s\w+]+
+##!> define quotes [\"'`]\s*
+##!> define digits \d+
+##!> define space \s+
+##!> define optional_space \s*
+
+##!> assemble
+  and
+  nand
+  or
+  xor
+  xxor
+  div
+  like
+  between
+  not
+  ##!=< logical_operators
+##!<
+
+
+##!> assemble
+  &
+  <
+  >
+  ^
+  =
+  ->
+  \*
+  /
```

Comment:
`->`, `*`, `/` are additions right? The weren't part of the regex before, AFAICT?

Reply:
Yes, but somehow they are out of sql operators in general. IMHO, having half of the operators makes for an easy bypass condition. But then the regexp is never going to be the same.

Reply:
I think it's a good idea to add them. But maybe do it in a follow up PR, so it's also clear in the history when the change happened. It makes our life easier if the switch to regex-assembly is 1:1.